rubocop-airbnb/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.0.3
```

Comment:
version numbers are only decided at publish time. (also, adding support for a rubocop version is at least semver-minor, not patch)
```suggestion
# Unreleased
```

Reply:
You're right, done 👍